api/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
Hmm... I think the change to this file can probably be reverted since there are no changes to pyproject.toml?

Reply:
I copied the main branch in, committed, then merged with main since there were too many main merges between poetry updates to do a clean revert. Either way, should be fixed now.